MicrosoftFluentUI.podspec

```diff
@@ -278,7 +278,8 @@ Pod::Spec.new do |s|
     core_mac.platform = :osx
     core_mac.resource_bundle = { 'FluentUIResources-macos' => ["apple/Resources/**/*.{json,xcassets}",
                                                              "macos/FluentUIResources-macos/**/*.{storyboard,xib,xcassets,strings,stringsdict}"] }
-    core_mac.source_files = ["macos/FluentUI/Core/**/*.{swift,h}"]
+    core_mac.source_files = ["macos/FluentUI/Core/**/*.{swift,h}",
+                              "macos/FluentUI/Extensions/**/*.{swift,h}"]
```

Comment:
By adding "Extensions" to the core subspec we're adding the Extensions folder as a dependency of other subspecs that don't need it (increasing the footprint for the consuming app).

We should create granular subspecs on folders that is more specific to the functionality that was added (color and appearance).

Maybe even splitting the file into NSAppearance and Color?

Reply:
Thanks for heads up. I agree with reducing efforts to lower footprint. I have broken down functionalities into multiple spec. Let me know your thoughts.